node/src/database/rocksdb.rs

```diff
@@ -757,6 +757,26 @@ impl<'db, DB: DBAccess> Metadata for DBTransaction<'db, DB> {
     }
 }
 
+impl<'db, DB: DBAccess> DBTransaction<'db, DB> {
+    /// A thin wrapper around inner.put_cf that calculates a db transaction
+    /// disk footprint
+    fn put_cf<K: AsRef<[u8]>, V: AsRef<[u8]>>(
+        &self,
+        cf: &impl AsColumnFamilyRef,
+        key: K,
+        value: V,
+    ) -> Result<()> {
+        let kv_size = key.as_ref().len() + value.as_ref().len();
+        self.inner.put_cf(cf, key, value)?;
+        *self.cumulative_inner_size.borrow_mut() += kv_size;
+        Ok(())
+    }
+
+    pub fn get_size(&self) -> usize {
+        *self.cumulative_inner_size.borrow()
```

Comment:
Is this updated when deleting data from db?

Reply:
Not related to deleted data at all. It's just the cumulative size of the a successfully committed/put data. (it's returned only by `Ledger::store_block` API

Reply:
That's ok, we just need to keep in mind this definition, otherwise we'll start asking why counter is different (if any node apply a fallback)